server/sonar-server/src/main/java/org/sonar/server/issue/filter/IssueFilterParameters.java

```diff
@@ -70,6 +70,8 @@
   public static final String ASC = "asc";
   public static final String IGNORE_PAGING = "ignorePaging";
 
+  public static final String FACET_ASSIGNED_TO_ME = "assigned_to_me";
```

Comment:
This constant should not be defined here, as this facet is not reserved to issue filters.